src/services/DocumentService.js

```
@@ -83,6 +85,12 @@ class DocumentService {
 			});
 	}
 
+	updateDocumentMetadata(recordId, jsonFHIR, tags = []) {
+		// don't update the content as it is internally handled by SDK
+		if (jsonFHIR.content) delete jsonFHIR.content;
```

Maybe we should throw an error here, to tell the user that he he updates the files wrong. 
But for now that's fine.